### ISSUE TITLE
Add option to exclude limited relationships from referential integrity violation analysis

### ISSUE THREAD

**Issue by albertospelta** [sql-bi/VertiPaq-Analyzer#132]
Currently, the referential integrity violation analysis includes all types of relationships - both regular and limited. This can lead to errors when limited relationships that span different data islands (cross-island relationships) generates queries that exceed row limits due to Direct Query mode (see https://github.com/sql-bi/Bravo/issues/798).

To address this, we propose adding a new option to specify the analysis mode, such as `Regular Only`(default) and `Full` which will retain the current behavior by including all relationships.

**Comment by marcosqlbi** [comment]
I completely agree, it's a good idea, and it should help get fewer errors with the default configuration.
